provisioning/device/src/TransportSettings/ProvisioningClientMqttSettings.cs

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Net.Security;
+
+namespace Microsoft.Azure.Devices.Provisioning.Client
+{
+    /// <summary>
+    /// Contains MQTT transport-specific settings for the device and module clients.
+    /// </summary>
+    public class ProvisioningClientMqttSettings : ProvisioningClientTransportSettings
+    {
+        private const bool DefaultCleanSession = false;
+        private readonly TimeSpan DefaultKeepAlive = TimeSpan.FromMinutes(2);
+        private const QualityOfService DefaultPublishToServerQoS = QualityOfService.AtLeastOnce;
+        private const QualityOfService DefaultReceivingQoS = QualityOfService.AtLeastOnce;
+
+        /// <summary>
+        /// Creates an instance of this class.
+        /// </summary>
+        /// <param name="transportProtocol">The transport protocol; defaults to TCP.</param>
+        public ProvisioningClientMqttSettings(ProvisioningClientTransportProtocol transportProtocol = ProvisioningClientTransportProtocol.Tcp)
+        {
+            Protocol = transportProtocol;
+            CleanSession = DefaultCleanSession;
```

Comment:
You could assign these inline (on the property definition). I think it improves code readability because assignments are local.